L1Trigger/VertexFinder/src/classes_def.xml

```diff
@@ -0,0 +1,33 @@
+<lcgdict>
+  <class name="l1tVertexFinder::InputData"/>
```

Comment:
All of these dictionaries should be declared as transient because they are defined in a non-DataFormat package.